src/components/Modals/DeleteCourseModal.vue

```diff
@@ -35,19 +35,19 @@ export default defineComponent({
   },
   emits: {
     'update:modelValue': (value: boolean) => typeof value === 'boolean',
-    'close-reset-modal': (value: boolean) => typeof value === 'boolean',
+    'close-delete-course-modal': (value: boolean) => typeof value === 'boolean',
   },
   methods: {
     closeCurrentModal(): void {
       this.$emit('update:modelValue', false);
     },
-    resetClicked(): void {
```

Comment:
I think you're going to have some merge conflicts here because of my modal fixes PR. I changed some things from the presentation, so let me know if you need any help resolving!